lib/srv/db/common/kerberos/client.go

```diff
@@ -46,20 +45,19 @@ type ClientProvider interface {
 	GetKerberosClient(ctx context.Context, ad types.AD, username string) (*client.Client, error)
 }
 
-func NewClientProvider(authClient windows.AuthInterface, dataDir string) ClientProvider {
-	return newClientProvider(authClient, dataDir)
+// NewClientProvider returns new instance of ClientProvider.
+func NewClientProvider(authClient windows.AuthInterface, dataDir string, logger *slog.Logger) ClientProvider {
```

Comment:
will this break e?

Reply:
It will, but I've raised the fix already: https://github.com/gravitational/teleport.e/pull/6518